utils/build/docker/python/flask/app.py

```diff
@@ -24,8 +22,6 @@
     host="postgres", port="5433", user="system_tests_user", password="system_tests", dbname="system_tests",
 )
 
-logging.basicConfig(level=logging.DEBUG)
```

Comment:
do we still want logging, just the default warning level or something?

Reply:
Default is error. For automatic tests on github action, we probably prefer to be sure that there is no flaky tests at all. But on local testing, this could be added back.